rocketmq-spring-boot/src/main/java/org/apache/rocketmq/spring/annotation/RocketMQMessageListener.java

```diff
@@ -143,6 +143,12 @@
      * The namespace of consumer.
      */
     String namespace() default "";
+    
+    /**
+     * The consumeMessageBatchMaxSize of consumer.
+     * @return
+     */
+    int consumeMessageBatchMaxSize() default 0;
```

Comment:
why consumeMessageBatchMaxSize default is zero and delayLevelWhenNextConsume  default is 1024 ？